NGitLab.Mock/GitLabServer.cs

```diff
@@ -130,11 +132,25 @@ public void Dispose()
 
         internal int GetNewResourceStateEventId() => Interlocked.Increment(ref _lastResourceStateEventId);
 
+        internal string GetNewRunnerToken() => MakeToken(Convert.ToString(Interlocked.Increment(ref _lastTokenId)));
+
+        internal string GetNewRegistrationToken() => MakeRegistrationToken(Convert.ToString(Interlocked.Increment(ref _lastRegistrationTokenId)));
+
         internal string MakeUrl(string relativeUrl)
         {
             return new Uri(Url, relativeUrl).AbsoluteUri;
         }
 
+        internal string MakeToken(string id, string prefix = "")
+        {
+            return prefix + id.PadLeft(20, '0');
+        }
+
+        internal string MakeRegistrationToken(string id)
+        {
+            return MakeToken(id, "GR1348941");
```

Comment:
```suggestion
            return MakeToken(id, "GR1234567A");
```

Reply:
The prefix I used is the real prefix used by GitLab for every registration token. I think it can help to directly recognize it

Reply:
https://gitlab.com/gitlab-org/gitlab/-/issues/388379